docs/IE_PLUGIN_DG/Doxyfile

```diff
@@ -780,10 +785,16 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = . \
-						 ../../src/common/transformations/include/transformations \
-						 ../../src/inference/dev_api \
-						 ../../src/common/itt/include/openvino
+INPUT                  = "@DOCS_BUILD_DIR@/openvino/docs/IE_PLUGIN_DG/" \
+                         "@OpenVINO_SOURCE_DIR@/src/common/transformations/include/transformations" \
```

Comment:
use `src/common/transformations/include/` w/o last `transformations`

It should fix the transformations in developer docs.